test/unit/callbacks_and_delays_test.lua

```diff
@@ -22,6 +22,13 @@ end)
 
 g.before_each(function(cg)
     cg.task_name = 'test'
+
+    local space = cg.space
+    local total = 10
+    for i = 1, total do
+        space:insert({i, tostring(i)})
+    end
+    t.assert_equals(space:count{}, total)
```

Comment:
This is a code style fix.
```suggestion
    t.assert_equals(space:count(), total)
```